charts/pravega/README.md

```diff
@@ -12,9 +12,9 @@ This chart creates a [Pravega](https://github.com/pravega/pravega) cluster in [K
   - Helm 3.2.1+
   - An existing Apache Zookeeper 3.6.1 cluster. This can be easily deployed using our [Zookeeper Operator](https://github.com/pravega/zookeeper-operator)
   - An existing Apache Bookkeeper 4.9.2 cluster. This can be easily deployed using our [BookKeeper Operator](https://github.com/pravega/bookkeeper-operator)
-  - Pravega Operator. You can install it using its own [Helm chart](https://github.com/pravega/pravega-operator/tree/master/charts/pravega-operator)
-
-## Installing the Chart
+  - Pravega Operator. Please refer [this](https://github.com/pravega/pravega-operator/blob/master/charts/pravega-operator/README.md)
+  - LongTerm Storage ([options for long term storage](doc/longtermstorage.md))
```

Comment:
This link is broken. Does it need to be:
`doc/longtermstorage.md` -> `../../doc/longtermstorage.md`?

Reply:
fixed